Angular/AsignacionAngular/RunForestRun/src/app/main-content/main-content.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { AgentServiceService} from './agent-service.service';
+import { Observable } from 'rxjs';
+
+
+@Component({
+  selector: 'app-main-content',
+  templateUrl: './main-content.component.html',
+  styleUrls: ['./main-content.component.scss']
+})
+export class MainContentComponent implements OnInit {
+    agents$: Object;
+    //variable for the filter function
+    agentFilter: any = {name: ''};
+
+  constructor(private agent_service: AgentServiceService) { }
+
+  ngOnInit() {
+    this.agent_service.getAgents().subscribe(
+      data => this.agents$ = data
+    )
+  }
+
+
+  modelChanged(e)  {
```

Comment:
Always comment functions that you create, put what the functions do, if it has parameters it is and it's type, if it returns a value what that value means and it's type

If you use some code editor like Visual Code you can download plugins that help you with what the comment of the functions should have